### DISCUSSION TITLE
ebooks/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/pengzhanbo.cn#45] [Blog-Comment]
# ebooks/

HTML
推荐 《Head First HTML与CSS（第2版）》 - 初学者入门; 
推荐 《HTML5+JavaScript动画基础》 - 动画基础入门; 
过时 《HTML5权威指南》 - 初学者入门; 
过时 《HTML5与CSS3权威指南》 - 初学者入门; 
过时 《HTML5程序设计》 - 初学者入门; 
过时 《HTML5揭秘》 - ...

https://pengzhanbo.cn/ebooks/

**Comment by LuckyQ7** [comment]
谢谢分享  很有用
